godtools/XMLContent/KGP/kgp-5.xml

```diff
@@ -100,7 +99,7 @@
                 </content:text>
             </paragraph>
             <paragraph>
-                <content:button type="event" value="kgp-6">
+                <content:button type="event" value="kgp-6,send-information-modal">
```

Comment:
I think setting a separate close listener might work better. I'm more concerned about overloading the event and making it mean different things based on current application state.

so, something like: `<modal listeners="send-information-modal" dismiss-listeners="send-information-model-close">`